common/sync_package.py

```diff
@@ -85,18 +85,30 @@ def get_version_tag():
 
 
 def update_libinfo(args):
-    _ , local_ver = get_version_tag()
+    pub_ver, local_ver = get_version_tag()
+
+    # Set the version to be compliant with what is accepted for
+    # PyPI. A valid version string looks like "0.8.dev1473"
+    package_ver = pub_ver if args.use_public_version else local_ver
 
     update(
         os.path.join(args.src, "python", "tvm", "_ffi", "libinfo.py"),
-        [("(?<=__version__ = \")[^\"]+", local_ver)],
+        [("(?<=__version__ = \")[^\"]+", package_ver)],
         args.dry_run
     )
 
 
 def update_setup(args, package_name):
+    pub_ver, local_ver = get_version_tag()
+
+    # Set the version to be compliant with what is accepted for
+    # PyPI. A valid version string looks like "0.8.dev1473"
+    package_ver = pub_ver if args.use_public_version else local_ver
+    package_ver = '"{0}"'.format(package_ver)
```

Comment:
```suggestion
    package_ver = f'"{package_ver}"'
```
nit: f-strings

Reply:
Crashes on mac.

Reply:
Are you using the OSX's default Python2? f-strings have been in Python since 3.6 and should be okay

Reply:
It seems Python 2 was being used... so I'll try to fix it in https://github.com/tlc-pack/tlcpack/pull/106